markdown/blog/gravitee-sponsorship-2025.md

```diff
@@ -0,0 +1,65 @@
+type: Communication
+tags:
+  - Project Status
+  - Marketing
+cover: /img/posts/marketing-images/gravitee-community.webp
+authors:
+  - name: Atinuke Oluwabamikemi Kayode
+    photo: /img/avatars/bami.webp
+    link: https://www.linkedin.com/in/atinuke-oluwabamikemi-kayode-5b838b1b7/
+    byline: AsyncAPI Community Marketing Specialist
+excerpt: 'Gravitee returns as a Platinum Sponsor, strengthening the AsyncAPI community and conference across 2025.'
+---
+
+
+We’re happy to have Gravitee return as a platinum sponsor for both the AsyncAPI community and the AsyncAPI Conferences in 2025! Their renewed partnership is not only a show of continued support but also a reflection of their belief in the power of open standards, community-driven innovation, and modern API ecosystems.
+Gravitee has long been part of AsyncAPI’s journey, not just as a sponsor but as an adopter, contributor, and champion of event-driven architectures (EDAs) that are secure, discoverable, and truly developer-first.
+
+## A Shared Vision: Open Collaboration, Stronger Infrastructure
+
+AsyncAPI’s mission is to make working with event-driven systems as seamless as it is working with REST. That mission only becomes a reality when organizations across industries join forces to push it forward sharing knowledge, building tooling, contributing to the spec, and supporting our diverse global community.
+Gravitee’s sponsorship is rooted in this shared vision. They understand that success in the API economy depends not only on robust tooling but on shared specifications and interoperable ecosystems. That’s why Gravitee has made AsyncAPI an essential part of its architecture, empowering teams to define, document, and govern their asynchronous APIs just as easily as their REST endpoints.
+
+By adopting and championing the AsyncAPI specification, Gravitee empowers teams to seamlessly design, manage, and secure asynchronous APIs such as Kafka, alongside traditional REST APIs. This approach enhances discoverability, governance, and observability across modern distributed systems, helping organizations unlock the full potential of event-based communication.
+This alignment has made Gravitee a natural supporter of AsyncAPI’s mission and growth.
+
+## More Than Sponsorship: Investing in Community-Driven Growth
+
+Gravitee’s renewed platinum sponsorship isn’t just a logo on a banner, it’s a commitment to sustainable, community-first growth. In 2024, their support helped us organize inclusive, impactful gatherings like AsyncAPI Conf Online and the Conference on Tour series now branded as the AsyncAPI Conference.
+In 2025, their contribution helps us go even bigger, with in-person conferences in Lagos, Munich, Bangalore, London, Paris, and Online each designed to bring the AsyncAPI community closer together across regions and time zones. Gravitee’s support enables:
+
+- Operational and travel cost for events
+- Broader outreach and visibility in growing async API communities around the world
```

Comment:
The word `async` confuses a bit and can mistakenly be treated like a typo of the organization name. Which one is it?
> in growing AsyncAPI communities around the world

or 

> in growing asynchronous API-related communities around the world

Reply:
The sentence needs to have clarity then to avoid such confusion